main.go

```diff
@@ -482,6 +498,9 @@ func Process(s *State) {
 	processorGroup.Wait()
 	close(resultChan)
 	printerGroup.Wait()
+	if s.OutputFileName != "" {
```

Comment:
Yeah mate, please commit changes to the same branch, and just push them up. The PR will update.